packages/rum-core/test/benchmarks/capture-navigation.bench.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+import { captureNavigation } from '../../src/performance-monitoring/capture-navigation'
+import Transaction from '../../src/performance-monitoring/transaction'
+import { PAGE_LOAD } from '../../src/common/constants'
+
+suite('CaptureNavigation', () => {
+  const options = {
+    managed: true,
```

Comment:
I think being managed or unmanaged doesn't affect these benchmarks, isn't that right?

Reply:
Yes, kept it explicitly as `managed` since `page-load` is a managed transaction.

Reply:
But, `page-load` being managed doesn't play a part in this benchmark, we could have a unmanaged transaction with the type `page-load` and the result of this benchmark would be the same.

Reply:
I can remove the flag.